### ISSUE TITLE
Configuring VGSCollect with VGSCollect.Builder crashes when setting custom hostname

### ISSUE THREAD

**Issue by rubina-shakhkyan** [verygoodsecurity/vgs-collect-android#145]
**Describe the bug**

**To Reproduce**
```
vgsForm = new VGSCollect.Builder(this.getCurrentActivity(), id)
                .setEnvironment(environment)
                .setHostname(hostname)
                .create();
```
This crashes on create, when running this client.enqueue(r) in VGSCollect.kt with `java.lang.NoSuchFieldError` exception. (line 647)
**Expected behavior**
I need to configure the VGSCollect sdk with builder in order to set a custom hostname.

**Screenshots**

**Environment (please complete the following information):**
 - SDK Version 1.6.1
 - Device: Pixel 4 API 29

**Comment by Endorf** [comment]
Thank you for the issue. We are investigating it.

**Comment by Endorf** [comment]
Hello! Do you have any updates? 

We couldn't reproduce your issue. So, I have several additional questions.
- Did you run the project from our repository or from your own?
- Did you try to run the [RN-Demo](https://github.com/verygoodsecurity/vgs-collect-android-react-native) by following our guide in ReadMe? 
- Could you try to clean the application and recompile all sources? I'm afraid that some sources have remained in a project when you built a new version.
- Do you use obfuscation in your project?

**Comment by rubina-shakhkyan** [comment]
The issue was fixed by upgrading Kotlin to 1.4.10 and okhttp to 4.9.0 fixed it.
